operator/v1/types_ingress.go

```diff
@@ -545,6 +545,21 @@ type AWSLoadBalancerParameters struct {
 	//
 	// +optional
 	NetworkLoadBalancerParameters *AWSNetworkLoadBalancerParameters `json:"networkLoadBalancer,omitempty"`
+
+	// subnets specifies the subnets for the load balancer to route
+	// traffic to. The subnets may be specified by either their
+	// ID or name.
+	// In order for the load balancer to be provisioned with subnets:
+	// * Each subnet must exist.
+	// * Each subnet must be from a different availability zone.
+	// * The load balancer service must be recreated to pick up new values.
+	//
+	// If omitted, the subnets will be auto-discovered per availability zone.
+	// When subnets are auto-discovered, they do not propagate to these lists.
+	//
+	// +optional
+	// +openshift:enable:FeatureGate=IngressControllerLBSubnetsAWS
+	Subnets AWSSubnets `json:"subnets,omitempty"`
```

Comment:
Omitempty does not work with structs that are not pointers, so a json marshal of this would result in failing your CEL rule on 577. This either needs to be a pointer to the struct, or that rule needs to be removed